webapp/src/app/ui/pull-request-widget/pull-request-widget.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="border border-gray-300 rounded-lg p-4 w-72">
+  <div class="flex justify-between items-center mb-2 text-xs text-gray-500">
+    <span class="font-bold">{{ pullRequest().repository.name }} #{{ pullRequest().number }} on {{ pullRequest().createdAt }}</span>
+    <span class="flex items-center space-x-2">
+      <span class="text-green-600 font-bold">+{{ pullRequest().additions }}</span>
```

Comment:
Here you should be using the GitHub color, inspect said they are using `color-fg-success` this maps to `text-github-success-foreground` on our side I believe.